miniwin/miniwin/src/include/miniwin_d3drmmesh_p.h

```diff
@@ -2,9 +2,20 @@
 
 #include "miniwin_d3drmobject_p.h"
 
+#include <unordered_map>
+
+struct MeshGroup {
```

Comment:
Having the API explicitly grouping everything by mesh greatly simplifies things.
We can upload the meshes as a single buffer, and do N render calls with only texture/material changed.

Reply:
Is that a thumbs up to my change or a request for more changes?

Reply:
Just a side note :) Looks like you can add vertices using `IDirect3DRMMesh::SetVertices`. So it's not possible to unify them in one buffer.